Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -57,42 +57,61 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .AddParameter("object", _("Shape Painter object"), "Drawer")
       .AddParameter("expression", _("X position of center"))
       .AddParameter("expression", _("Y position of center"))
-      .AddParameter("expression", _("Radius ( in pixels )"))
+      .AddParameter("expression", _("Radius (in pixels)"))
       .SetFunctionName("DrawCircle")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
   obj.AddAction("Line",
                 _("Line"),
                 _("Draw a line on screen"),
                 _("Draw from _PARAM1_;_PARAM2_ to _PARAM3_;_PARAM4_ a line "
-                  "(thickness : _PARAM5_) with _PARAM0_"),
+                  "(thickness: _PARAM5_) with _PARAM0_"),
                 _("Drawing"),
                 "res/actions/line24.png",
                 "res/actions/line.png")
 
+      .SetHidden()
       .AddParameter("object", _("Shape Painter object"), "Drawer")
       .AddParameter("expression", _("X position of start point"))
       .AddParameter("expression", _("Y position of start point"))
       .AddParameter("expression", _("X position of end point"))
       .AddParameter("expression", _("Y position of end point"))
-      .AddParameter("expression", _("Thickness ( in pixels )"))
+      .AddParameter("expression", _("Thickness (in pixels)"))
       .SetFunctionName("DrawLine")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
+  
+   obj.AddAction("LineV2",
+                _("Line V2"),
+                _("Draw a line V2 on screen"),
+                _("Draw from _PARAM1_;_PARAM2_ to _PARAM3_;_PARAM4_ a line V2 "
+                  "(thickness: _PARAM5_) with _PARAM0_"),
+                _("Drawing"),
+                "res/actions/line24.png",
+                "res/actions/line.png")
+
+      .AddParameter("object", _("Shape Painter object"), "Drawer")
+      .AddParameter("expression", _("X position of start point"))
+      .AddParameter("expression", _("Y position of start point"))
+      .AddParameter("expression", _("X position of end point"))
+      .AddParameter("expression", _("Y position of end point"))
+      .AddParameter("expression", _("Thickness (in pixels)"))
+      .SetFunctionName("drawLineV2")
+      .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
   obj.AddAction("Ellipse",
                 _("Ellipse"),
                 _("Draw an ellipse on screen"),
-                _("Draw at _PARAM1_;_PARAM2_ an ellipse of half width _PARAM3_ and half height _PARAM4_ "
+                _("Draw at _PARAM1_;_PARAM2_ an ellipse of width _PARAM3_ and height _PARAM4_ "
```

Comment:
There is a useless extra space at the end of this sentence and some others too.

Reply:
```
_("Draw at _PARAM1_;_PARAM2_ an ellipse of width _PARAM3_ and height _PARAM4_ "
 "with _PARAM0_"),
```
?
Because other extension was like this before, on two lines.

Reply:
Ah my bad this one is fine!